inc/admin/ui/options.php

```diff
@@ -24,7 +24,7 @@ function wpt_customofile_tools_page() {
 	<div class="wpt-customofile-options-page wrap">
 		<header class="wpt-header">
 		<div>
-		<img src="<?php echo WPT_CUSTOMOFILE_IMG_URL . 'wpt-logo.svg'; ?>" />
+		<img src="<?php echo esc_url( WPT_CUSTOMOFILE_IMG_URL . 'wpt-logo.svg' ); ?>" />
 		<a href="https://www.transifex.com/wp-translations/" class="wpt-transifex-link alignright"><?php esc_html_e( 'Join us on transifex', 'wpt-custom-mo-file' ); ?></a>
```

Comment:
This line should be changed as we don't use Transifex anymore… Looking for Translation Services…
Create an issue if needed.

Reply:
Changed to an optional Call to Action on https://github.com/WP-Translations/wpt-custom-mo-file/pull/22/commits/5ccd03f07ad4f3b856e43b408390c5827b82d23d